### ISSUE TITLE
Modelling of two drugs

### ISSUE THREAD

**Issue by tamas-ferenci** [nlmixrdevelopment/nlmixr#141]
It might be a very uninformed question (I have no background in PK itself), but is it possible to account for two drugs in the same model with `nlmixr`? In our particular example, we have 4 compartments, the first drug is injected regularly to the first compartment, the second drug goes to the second compartment, but they can't be merged (the differential equations of the remaining two compartments depend separately on them, in a quite complex way).

**Comment by mattfidler** [comment]
Yes @tamas-ferenci , you can account for the PK of the two compartments in the same model as nlmixr.

You specify differential equations:

```R
model({
...
d/dt(drug1)=....
d/dt(drug2)=....
...
cd1 = drug1/v1
cd2 = drug2/v2

cd1 ~ prop(prop.err) | drug1
cd2 ~ prop(prop.err2) | drug2
```
Notice the condition is the compartment item that the drug will be modeled in.  The CMT will specify the compartment number for whatever PK is being modeled.

**Comment by tamas-ferenci** [comment]
Thank you very much @mattfidler !

There is one minor point in which I'm not entirely sure. Your code seems to suggest (but I might be wrong!) that I can measure those two compartments that are dosed with the drug. This is, unfortunately, not our case. What we have is:

```
    d/dt(x1) = ...
    d/dt(x2) = ...
    d/dt(x3) = ...
    d/dt(x4) = ...
    
    Volume = x1 + x2
    Volume ~ add(add.err)
```

with the first drug going to `x3`, the second going to `x4`.

What should be the syntax in this case...?

**Comment by mattfidler** [comment]
This issue should explain the syntax

https://github.com/nlmixrdevelopment/nlmixr/issues/140#issuecomment-473019758

**Comment by tamas-ferenci** [comment]
I really tried to do my best, but I feel I'm still lost, sorry :(

It might be that I simply misunderstand you, but...

```
Plasma_Exo1H_total_nM= (CPL_Exo1H + CPL_Exo1TL1AH + CPL_Exo1MTL1AH)*(1e+09);
      CPmAbT=Plasma_Exo1H_total_nM; # What I want to model
  CPmAbT ~  prop(prop.err) | CPL_Exo1H # Model assuming proportional error w/CMT=CPL_Exo1H
# Note that CPL_Exo1H is just an indicator of where the response is found.
```

...in my example, the response isn't found in any compartment (rather, it is the sum of two compartments).

**Comment by mattfidler** [comment]
The response is not currently related to the compartment.  The compartment is just a surrogate for where the effect is occurring.  

This is being refactored because I think it is hard to follow.

```R
d/dt(cmt1)= # Ode for cmt1
d/dt(cmt2)= # Ode for cmt2

y1 = cmt1/v1 + cmt2/v2

y1 ~ add(add.err) | cmt1 # cmt1 is only a surrogate for the effect and does not mean that the cmt1 levels relate
```

**Comment by tamas-ferenci** [comment]
Thank you very much! I simply checked that the result is the same with `|cmt1` and `|cmt2`. Nevertheless I agree that a more logical syntax here would be a nice addition. Thank you again!

**Comment by mattfidler** [comment]
Great
